packages/extras/src/lib/types/components.ts

```diff
@@ -146,3 +154,20 @@ export type EnvironmentProperties = {
    */
   encoding?: TextureEncoding
 }
+
+export type SparklesProperties = Omit<Points, 'scale'> & {
+  /** Number of particles (default: 100) */
+  count: number
+  /** Speed of particles (default: 1) */
+  speed: number | Float32Array
+  /** Opacity of particles (default: 1) */
+  opacity: number | Float32Array
+  /** Color of particles (default: 100) */
+  color: ColorRepresentation | Float32Array
+  /** Color of particles (default: 100) */
+  size: number | Float32Array
+  /** The space the particles occupy (default: 1) */
+  scale: number | [number, number, number] | Vector3
+  /** The space the particles occupy (default: 1) */
+  noise: number | [number, number, number] | Vector3 | Float32Array
+}
```

Comment:
The types are now located in a separate file next to the component itself, remove the types here.

Reply:
(Other components will follow that principle too in the future)